### DISCUSSION TITLE
Make logs accessible within iOS without needing to access it via tethering

### DISCUSSION THREAD

**Discussion by fyyrraaa** [ppy/osu#19156] [Ideas] [answered]
One thing that somewhat bugs me is that if I want to retrieve logs from my iPad or my iPhone, I would have to tether it to my computer whereas in Android (according to the instructions) it would've been accessible via file managers.

It might be nice to have it accessible via the Files app as either a File Provider (ie. appearing under the Locations header in the sidebar) or via the On My iPad/iPhone location.

**Comment by peppy** [comment, marked as answer]
We'll eventually add a way to submit a bug report from within the app.
